pkg/chart/chart.go

```diff
@@ -117,18 +129,19 @@ func (ch *Chart) AppVersion() string {
 	return ch.Metadata.AppVersion
 }
 
-// CRDs returns a list of File objects in the 'crds/' directory of a Helm chart.
-func (ch *Chart) CRDs() []*File {
-	files := []*File{}
+// CRDs returns a list of CRD objects in the 'crds/' directory of a Helm chart & subcharts
+func (ch *Chart) CRDs() []CRD {
```

Comment:
So technically speakign we cannot break the API for this, and changing the function's return type would break the API.

I would recommend just creating a function named something like `CRDObjects()` and returning the `[]CRD` from that, and figuring another way to keep this function basically as it was.

Then flag this function as DEPRECATED and we can remove it in Helm 4.

Reply:
Thanks, I didn't realize these APIs were set for other usage. I'll adjust accordingly shortly.

Reply:
@technosophos how's this update for you?